libs/remix-lib/src/types/ICompilerApi.ts

```diff
@@ -24,7 +24,7 @@ export interface ICompilerApi {
 
     onCurrentFileChanged: (fileName: string) => void
     // onResetResults: () => void,
-    onSetWorkspace: (workspace: any) => void
+    onSetWorkspace: (isLocalhost: boolean, workspaceName: string) => void
```

Comment:
The old argument contains `isLocalhost` and `workspaceName` as properties already. Is there any reason why you seperated them into individual arguments?

Reply:
I don't remember details but there was. maybe because the name localhost doesn't mean it is. i should look closer

Reply:
besides i always pass as little as is necessary. :))